test/util/assertAriaOwns.js

```diff
@@ -0,0 +1,48 @@
+const assert = require('assert');
+
+/**
+ * Confirm the aria-owns element.
+ *
+ * @param {obj} t                  - ava execution object
+ * @param {String} elementSelector - the element with aria-owns set
+ */
+
+module.exports = async function assertAriaOwns(t, elementSelector) {
```

Comment:
I appreciate this addition to the helper functions! But I noticed you don't actually use it in your test...? You could use it for the `menubar-menuitem-aria-owns` test